modules/rosa-rn-acct-management.adoc

```diff
@@ -0,0 +1,27 @@
+:_module-type: CONCEPT
+// Module included in the following assemblies:
+//
+// rosa_release_notes/rosa-rn-new-features.adoc
+
+[id="rosa-rn-acct-management_{context}"]
+= Account management
+
+[id="rosa-rn-billing_{context}"]
+== Billing
+
+{product-title} (ROSA) is billed through Amazon Web Services (AWS) based on usage of AWS components used by the service, such as load balancers, storage, EC2 instances, and so on.
+
+[id="rosa-rn-self-service_{context}"]
+== Cluster self-service and consoles
+
+Customers can create, scale, and delete their clusters either from OpenShift Cluster Manager (OCM) or the `rosa` CLI utility.
+
+[id="rosa-rn-compute_{context}"]
+== Compute
+
+ROSA supports single availability zone and multiple availability zone clusters. Worker nodes must all be the same instance type and size within a single ROSA cluster.
+
+[id="rosa-rn-support_{context}"]
+== Support
+
+ROSA includes Red Hat Premium Support, which can be accessed by using the link:https://access.redhat.com/support[Red Hat Customer Portal] or through link:https://aws.amazon.com/premiumsupport/[AWS Support].
```

Comment:
I think as part of the process they need to create/have/identify their Red Hat account otherwise access won't be able to be granted within OCM.  I am not sure what the workflow is though if someone tries to create a ROSA cluster and does not have a red hat account. @acathrow should know.

Reply:
Since this is the Release Notes, I don't think we need to get into that level of granularity. Other documents provide the Support details, like the Service Definition, and Andy said not to tie the Support info to the account info. To use ROSA, they have to create an RH account to log in and use the service anyway, which is detailed in the user docs.

I'm leaving this as-is.